app/design/frontend/Magebit/learning/Magento_Catalog/templates/product/list/items.phtml

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * This file is part of the Magebit package.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magebit ProductAttributes
+ * to newer versions in the future.
+ *
+ * @copyright Copyright (c) 2024 Magebit, Ltd. (https://magebit.com/)
+ * @license   GNU General Public License ("GPL") v3.0
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+/* @var $block \Magento\Catalog\Block\Product\AbstractProduct */
+
+/** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+?>
+
+<?php
+switch ($type = $block->getType()) {
+    case 'related':
+        /** @var \Magento\Catalog\Block\Product\ProductList\Related $block */
```

Comment:
I'm not sure but I think these also could be moved to the first   <?PHP> and added to use, test it out if you can do it do it like that because that should be the correct way